site/src/components/RuntimeErrorState/RuntimeErrorState.tsx

```diff
@@ -1,125 +1,216 @@
-import Box from "@material-ui/core/Box"
+import Button from "@material-ui/core/Button"
 import Link from "@material-ui/core/Link"
 import { makeStyles } from "@material-ui/core/styles"
-import ErrorOutlineIcon from "@material-ui/icons/ErrorOutline"
-import { useEffect, useReducer, FC } from "react"
-import { mapStackTrace } from "sourcemapped-stacktrace"
+import RefreshOutlined from "@material-ui/icons/RefreshOutlined"
+import { BuildInfoResponse } from "api/typesGenerated"
+import { CopyButton } from "components/CopyButton/CopyButton"
+import { CoderIcon } from "components/Icons/CoderIcon"
+import { FullScreenLoader } from "components/Loader/FullScreenLoader"
+import { Stack } from "components/Stack/Stack"
+import { FC, useEffect, useState } from "react"
+import { Helmet } from "react-helmet-async"
 import { Margins } from "../Margins/Margins"
-import { Section } from "../Section/Section"
-import { Typography } from "../Typography/Typography"
-import {
-  createFormattedStackTrace,
-  reducer,
-  RuntimeErrorReport,
-  stackTraceAvailable,
-  stackTraceUnavailable,
-} from "./RuntimeErrorReport"
-
-export const Language = {
-  title: "Coder encountered an error",
-  body: "Please copy the crash log using the button below and",
-  link: "send it to us.",
-}
 
-export interface RuntimeErrorStateProps {
-  error: Error
-}
+const fetchDynamicallyImportedModuleError =
+  "Failed to fetch dynamically imported module" as const
```

Comment:
Why do we have to cast this `as const`?

Reply:
I don't remember, I think it was Co-pilot 🤔 I will remove it since does not influence anything